gradle/publish.gradle

```diff
@@ -0,0 +1,78 @@
+apply plugin: 'com.novoda.build-properties'
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+
+buildProperties {
+    cli {
+        using(project)
+    }
+
+    bintray {
+        def bintrayCredentials = {
+            return isDryRun() ?
+                    ['bintrayOrg': 'n/a', 'bintrayRepo': 'n/a', 'bintrayUser': 'n/a', 'bintrayKey': 'n/a'] :
+                    new File("${System.getenv('BINTRAY_PROPERTIES')}")
+        }
+        using(bintrayCredentials()).or(cli)
+        description = '''This should contain the following properties:
+                        - bintrayOrg: name of the Bintray organisation to deploy the artifacts to
+                        - bintrayRepo: name of the repo of the organisation to deploy the artifacts to
+                        - bintrayUser: name of the account used to deploy the artifacts
+                        - bintrayKey: API key of the account used to deploy the artifacts'''
+                .stripIndent()
+    }
+}
+
+
+def pomConfig = {
+    licenses {
+        license {
+            name "The Apache Software License, Version 2.0"
+            url "http://www.apache.org/licenses/LICENSE-2.0.txt"
+            distribution "repo"
+        }
+    }
+    developers {
+        developer {
+            id "novoda"
```

Comment:
Not sure about these ones 🤔

Reply:
I don't think this is needed, or at least it's not an information that I see in any of the pom files generated using the `bintray-release` plugin.

Reply:
Okay, let's remove `developers` and `scm` then 👍